auth/src/main/java/com/firebase/ui/auth/util/GoogleSignInHelper.java

```diff
@@ -26,7 +25,7 @@ public static GoogleSignInHelper getInstance(FragmentActivity activity) {
 
     public Task<Status> signOut() {
         final TaskCompletionSource<Status> statusTask = new TaskCompletionSource<>();
-        getConnectedApiTask().addOnSuccessListener(new OnSuccessListener<Bundle>() {
+        getConnectedApiTask().addOnSuccessListener(new ExceptionForwarder<Bundle>(statusTask) {
```

Comment:
I don't think this will work, it's still setting the `ExceptionForwarder` using `addOnSuccessListener` so even though it implements `onFailure` that will never be called.

You may want to make this an `onCompleteListener` instance instead so you'll get notified of success or failure.